spark/src/main/kotlin/com/adevinta/spark/components/tags/Tag.kt

```diff
@@ -0,0 +1,179 @@
+    ) {
+        ProvideTextStyle(
+            value = SparkTheme.typography.smallImportant,
+        ) {
+            Row(
+                Modifier
+                    .defaultMinSize(minHeight = MinHeight)
+                    .padding(vertical = VerticalPadding, horizontal = HorizontalPadding),
+                horizontalArrangement = Arrangement.spacedBy(LeadingIconEndSpacing, Alignment.Start),
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                if (leadingIcon != null) {
+                    CompositionLocalProvider(
+                        LocalContentColor provides colors.contentColor,
+                    ) {
+                        Icon(
+                            sparkIcon = leadingIcon,
+                            modifier = Modifier.size(LeadingIconSize),
+                            contentDescription = null, // The tag is associated with a mandatory label so it's okay
+                            tint = tint ?: LocalContentColor.current,
+                        )
+                    }
+                }
+                content()
+            }
+        }
+    }
+}
+
+@Immutable
+data class TagColors(
+    val backgroundColor: Color,
+    val contentColor: Color,
+)
+
+object TagDefaults {
+    internal val MinHeight = 20.dp
+
+    /**
+     * The outlined chip's border size
+     */
+    internal val OutlinedBorderSize = 1.dp
+
+    /**
+     * The size of a chip's leading icon
+     */
+    internal val LeadingIconSize = 16.dp
+
+    @Composable
+    fun tonalColors(
+        backgroundColor: Color = SparkTheme.colors.secondaryContainer,
+    ): TagColors {
+        return TagColors(
+            backgroundColor = backgroundColor,
+            contentColor = contentColorFor(backgroundColor = backgroundColor),
+        )
+    }
+
+    @Composable
+    fun filledColors(): TagColors = TagColors(
+        backgroundColor = SparkTheme.colors.primary,
+        contentColor = SparkTheme.colors.onPrimary,
+    )
+
+    @Composable
+    fun outlineColors(
+        contentColor: Color = SparkTheme.colors.tertiary,
+    ): TagColors = TagColors(
+        backgroundColor = Color.Transparent,
+        contentColor = contentColor,
+    )
+}
+
+private val LeadingIconEndSpacing = 4.dp
+
+/**
+ * The content padding used by a chip.
+ * Used as start padding when there's leading icon, used as eng padding when there's no
+ * trailing icon.
+ */
+private val HorizontalPadding = 8.dp
+
+/**
+ * The content padding used by a chip.
+ * Used as start padding when there's leading icon, used as eng padding when there's no
+ * trailing icon.
+ */
+private val VerticalPadding = 2.dp
+
+@Preview(
+    group = "Tags",
+    name = "Tag",
+)
+@Composable
+private fun SparkTagPreview() {
+    PreviewTheme {
+        val colors = TagDefaults.filledColors()
+        SparkTag(colors = colors) {
+            SlotArea(color = LocalContentColor.current) {
+                Text("À la une")
```

Comment:
Do we want to use french wording in previews?

Reply:
I ask Mathieu and it's okay for now since they don't have english guidelines for now